crates/sui-json-rpc/src/read_api.rs

```diff
@@ -1107,6 +1107,14 @@ fn convert_to_response(
     let mut response = SuiTransactionResponse::new(cache.digest);
     response.errors = cache.errors;
 
+    if opts.show_raw_input && cache.transaction.is_some() {
+        let sender_signed_data = cache.transaction.as_ref().unwrap().data();
```

Comment:
you prob thought of this already, is unwrap() safe here, add comments if so?

Reply:
yes this is safe. Will add a comment

Reply:
Yes because we checked `cache.transaction.is_some()` on line 1110